### PR TITLE
Updated deps and C API code for Math::Int64/128

### PR DESCRIPTION
**Pull request by autarch**

### CODE REVIEW

**Comment by autarch** [comment]
Travis is being weird but you can see that this branch passed here - https://travis-ci.org/maxmind/MaxMind-DB-Writer-perl/builds/53995344

File: c/perl_math_int128.c [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
 #include "EXTERN.h"
 #include "perl.h"
 #include "ppport.h"
 
-#if ((LONGSIZE >= 8) && (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 4)))
-
 #if ((__GNUC__ == 4) && (__GNUC_MINOR__ < 6))
 
-/* workaround for gcc 4.4/4.5 - see http://gcc.gnu.org/gcc-4.4/changes.html */
+/* workaroung for gcc 4.4/4.5 - see http://gcc.gnu.org/gcc-4.4/changes.html */
```
Comment (oschwald) [inline c/perl_math_int128.c]: I assume this typo is from upstream.

Comment (autarch) [inline c/perl_math_int128.c]: Yes.

Comment (autarch) [inline c/perl_math_int128.c]: This entire file is simply copied from upstream.

**Pull request merged**